pkg/task/common.go

```diff
@@ -49,6 +53,25 @@ type TLSConfig struct {
 	Key  string `json:"key" toml:"key"`
 }
 
+// Enable checks if TLS open or not
+func (tls *TLSConfig) Enable() bool {
```

Comment:
```suggestion
func (tls *TLSConfig) IsEnabled() bool {
```

The current name sounds like you can enable TLS with this method.